### PR TITLE
Download latest version (6.2.0) of PSDesiredStateConfiguration nuget package.

### PR DESCRIPTION
**Pull request by amitsara**
# PR Summary
Latest PSDesiredStateConfiguration nuget package contains fixes for DSC compilation issues on PowerShell core. This PR is to pick those changes.
All the PowerShell module files are signed in nuget package so PowerShell doesn’t need to sign them again. Removing PSDesiredStateConfiguration.psm1 signing.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 1 hour

**Review by adityapatwardhan — APPROVED** [review]

**Comment by TravisEz13** [comment]
@PoshChan Please get errors

**Comment by PoshChan** [comment]
@TravisEz13, I do not understand: Please get errors

Commands available in this repo for you:
  - `retry <target>` this will attempt to retry only the failed jobs for the target pipeline
  - `rebuild <target>` this will perform a complete rebuild of the target pipeline, `rerun` can be used in place of `rebuild`
    Supported values for \<target\> which can be a comma separated list are: `static`,`windows`,`macos`,`linux`,`all`
  - `get failures` this will attempt to get the latest failures for all of the target pipelines
  - `remind me in <value> <units>` this will create a reminder that will be posted after the specified duration
    \<value\> is a number, and \<units\> can be `minutes`, `hours`, or `days` (singular or plural)

**Comment by TravisEz13** [comment]
@PoshChan Please get failures

File: nuget.config [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <clear />
```
Comment (PoshChan) [inline nuget.config]: @TravisEz13, test results were not published for PowerShell-CI-install-ps at vstfs:///Build/Build/26452

Comment (PoshChan) [inline nuget.config]: @TravisEz13, your last commit had 3 failures in `PowerShell-CI-linux`
DSC MOF Compilation.Should be able to compile a MOF from a basic configuration
```powershell
Expected no exception to be thrown, but an exception "Errors occurred while processing configuration 'DSCTestConfig'." was thrown from /home/vsts/work/1/a/bins/publish/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.psm1:3710 char:5
    +     throw $ErrorRecord
    +     ~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 30
30: "@) | Should -Not -Throw
```
DSC MOF Compilation.Should be able to compile a MOF from another basic configuration
```powershell
Exception calling "Create" with "1" argument(s): "At line:5 char:17
+                 nxScript f1
+                 ~~~~~~~~
Undefined DSC resource 'nxScript'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 36
36:         [Scriptblock]::Create(@"

```
DSC MOF Compilation.Should be able to compile a MOF from a complex configuration
```powershell
Expected no exception to be thrown, but an exception "Errors occurred while processing configuration 'WordPressServer'." was thrown from /home/vsts/work/1/a/bins/publish/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.psm1:3710 char:5
    +     throw $ErrorRecord
    +     ~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1: line 161
161: "@) | Should -Not -Throw
```

Comment (PoshChan) [inline nuget.config]: @TravisEz13, test results were not published for PowerShell-CI-macos at vstfs:///Build/Build/26449

Comment (PoshChan) [inline nuget.config]: @TravisEz13, test results were not published for PowerShell-CI-static-analysis at vstfs:///Build/Build/26448

Comment (PoshChan) [inline nuget.config]: @TravisEz13, test results were not published for PowerShell-CI-windows at vstfs:///Build/Build/26451

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 hour ago

**Comment by TravisEz13** [comment]
@amitsara Please fix test failures

**Comment by amitsara** [comment]
Now $env:DSC_HOME is being used by psdesiredstateconfiguration.psm1 to point the base schema file. We cannot use it in the test. 
Please review my changes in test file. Test failures are fixed now. windows packaging is failing for some reason.

---
In reply to: [505648159](https://github.com/PowerShell/PowerShell/pull/9932#issuecomment-505648159) [](ancestors = 505648159)

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 1 hour.

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 hour. ago

**Pull request merged**